### PR TITLE
Added timezone to timestamps.

### PR DESCRIPTION
**Pull request by vruyr**

### CODE REVIEW

File: PlainTasks.py [hunk 1]
```diff
@@ -28,6 +28,14 @@
     import subprocess
 
 
+def tznow():
+    return datetime.now().replace(
```
Comment (vovkkk) [inline PlainTasks.py]: How it differs from `datetime.now(tz=timezone(datetime.fromtimestamp(0) - datetime.utcfromtimestamp(0)))`?

Comment (vruyr) [inline PlainTasks.py]: If the timezone is passed to datetime.datetime.new() and timezone info is supported for the platform (gettimeofday) resulting datetime object will a conversion from local timezone to the specified one. In contrast, datetime.datetime.replace() just changes the appropriate property.

https://docs.python.org/3.5/library/datetime.html#datetime.datetime.now

**Comment by vruyr** [comment]
> ST2 ImportError: cannot import name timezone
Oops! Implemented rudimentary timezone class for ST2.

**Comment by vovkkk** [comment]
Will you add this support to PlainTasksDates.py?

**Comment by vovkkk** [comment]
We already depends on dateutil, and pytz is availaible as dependency https://github.com/packagecontrol/pytz

**Comment by vruyr** [comment]
I might have some free time next week – will try to squeeze that in.

**Pull request merged**